src/components/RevokeContainer/RevokeDocumentDropZone/RevokeDocumentDropZone.tsx

```diff
@@ -49,33 +49,36 @@ export const RevokeDocumentDropZone: FunctionComponent<RevokeDocumentDropZone> =
         <IssueOrRevokeSelector />
       </div>
       <ContentFrame>
-        <h3 data-testid="revoke-title" className="mb-8 font-bold">
-          Revoke Document
-        </h3>
         <Card>
           <ProgressBar step={1} totalSteps={3} title="Upload Document" />
-          <div className="my-6 text-2xl">Upload Document</div>
+          <div data-testid="revoke-title" className="my-6 text-2xl">
+            Upload File
+          </div>
           <div {...getRootProps()} data-testid="revoke-dropzone">
             <input data-testid="revoke-document-drop-zone" {...getInputProps()} />
             <DropZone
               isDragActive={isDragActive}
-              error={Boolean(errorMessage || documentUploadState === DocumentUploadState.ERROR)}
+              error={
+                (errorMessage !== undefined && errorMessage.length > 0) ||
+                documentUploadState === DocumentUploadState.ERROR
+              }
             >
-              <img className="mb-12" src={"/dropzone-graphic.png"} />
               {documentUploadState === DocumentUploadState.LOADING && (
                 <div className="py-8 flex flex-col items-center" data-testid="dropzone-loader">
-                  <LoaderSpinner />
-                  <div className="mt-4 text-blue">Verifying Document</div>
+                  <LoaderSpinner primary="#3B8CC5" />
+                  <div className="mt-4 text-cerulean font-bold">Verifying Document</div>
                 </div>
               )}
               <>
+                {documentUploadState !== DocumentUploadState.LOADING && (
+                  <img className="mb-12" src={"/dropzone-graphic.png"} />
+                )}
                 {documentUploadState === DocumentUploadState.ERROR && (
-                  <div className="max-w-lg text-rose font-bold text-lg" data-testid="error-message">
-                    <p>{errorMessage ? errorMessage : "Error: File cannot be read"}</p>
-                    <p className="text-base text-gray-800 my-4 font-normal">Please try again</p>
+                  <div className="max-w-lg text-rose font-bold text-lg mb-4" data-testid="error-message">
+                    {errorMessage ? errorMessage : "File cannot be read. Please check that you have a valid file"}
                   </div>
                 )}
-                {documentUploadState === DocumentUploadState.INITIALIZED && (
+                {documentUploadState !== DocumentUploadState.LOADING && (
```

Comment:
whats wrong with the previous logic?

Reply:
this line should show regardless of whether the documentUploadState is in ERROR or INITIALISED, can refer to the figma file

Reply:
👌